Source/WebCore/rendering/updating/RenderTreeBuilder.cpp

```diff
@@ -942,10 +942,11 @@ RenderPtr<RenderObject> RenderTreeBuilder::detachFromRenderElement(RenderElement
     // So that we'll get the appropriate dirty bit set (either that a normal flow child got yanked or
     // that a positioned child got yanked). We also repaint, so that the area exposed when the child
     // disappears gets repainted properly.
+    bool shouldNotRepaint = child.previousSibling() && is<RenderMultiColumnSet>(child.previousSibling());
```

Comment:
I'd move this inside "if (!parent.renderTreeBeingDestroyed()" and skip both repaints.

Reply:
also is<> can take null_ptr